tools/generators/legacy/src/Generator/SetTargetConfigurations.swift

```diff
@@ -812,11 +812,7 @@ where Key == BuildSettingConditional, Value == [String: BuildSetting] {
         // TODO: If we ever add support for Universal targets we need to
         //   consolidate "ARCHS" to an `.any` conditional
 
-        let baseConditions = self.keys
-            .sorted()
-            .filter { $0 != .any }
-
-        let baseCondition = baseConditions.first!
+        let baseConditions = Set(keys.sorted().filter { $0 != .any })
```

Comment:
```suggestion
        let baseConditions = Set(keys).sorted().filter { $0 != .any })
```

I don't think `Set` guarantees order

Reply:
Removed the sort since we don't care about the order anymore.

Reply:
Makes sense